.github/workflows/ci.yaml

```diff
@@ -15,21 +15,82 @@ env:
   NODE_OPTIONS: --max_old_space_size=6144
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+permissions: {}
+
 jobs:
+  dedupe:
+    name: Deduplicate dependencies
+    # Skip unless this is a dependabot pull request
+    if: |
+      github.actor == 'dependabot[bot]' &&
+      startsWith(github.head_ref, 'dependabot/npm_and_yarn/')
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    outputs:
+      # Downstream jobs need to use this SHA to get the dedupe commit
+      sha: ${{ steps.get-sha.outputs.sha }}
+    steps:
+      - name: Check out files from GitHub
+        uses: actions/checkout@v3
```

Comment:
Pin the version with minor and patch

Reply:
I agree, but this is consistent with the rest of the frontend workflows and unrelated.  I can do a follow up PR with a global replace for all instances.

Reply:
See #14894.  If that is merged, I can quickly update here.

Reply:
Done.